std_msgs/Header header

string ctrl_state

int32 idMotion

bool jointPos_ok
bool cart_ok

JointJog joint_jog

float64[3] tool_position_xyz
float64[3] tool_orientation_rpy

